### DISCUSSION TITLE
Where are 'x' data stored in `ModelResult` object ?

### DISCUSSION THREAD

**Discussion by cgadal** [lmfit/lmfit-py#869] [Q&A] [answered]
For a `ModelResult` resulting from:

`result = gmodel.fit(y, params, x=x)`

How can I access the `x` data in `result` (`y` is stored in `result.data`)  ?

**Comment by newville** [comment, marked as answer]
@Cgadal the values that are passed to `Model.fit` as keyword/value arguments are stored in the dictionary `ModelResult.userkws`.  So, you would want ``result.userkws['x']``.

**Reply by cgadal** [reply]
Thanks! Should this be added to the documentation (https://lmfit.github.io/lmfit-py/model.html)?

**Reply by newville** [reply]
@Cgadal Oh, yes I see that it is not included in the list of attributes. I think it should be!  If you're willing to make such a change to the docs, that would be much appreciated. If not, let us know and we'll add it.

**Reply by cgadal** [reply]
@newville If you are not in a hurry, I'm willing to propose changes to the documentation. As I am just starting to use the `lmfit` package, I noticed a couple things that could be clarified !

**Reply by newville** [reply]
Sure, if you have proposed changes to the docs, you can make them anytime.
